src/pages/HomePage/components/Rating/Star/Star.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from '../Rating.module.sass'
+import StarFillIcon from 'remixicon-react/StarFillIcon'
+import StarLineIcon from 'remixicon-react/StarLineIcon'
+
+function Star({num}) {
+    return (
+        <div className={styles["rating-star"]}>
+            {
+                [1,2,3,4,5].map( (item, index) => {
```

Comment:
```suggestion
                [...Array(5).keys()].map((item, index) => {
```
Nó cũng có thể được viết như này nhé, trong trường hợp 1 mảng quá dài như `[1,2,3,4,5,6,7,8,9,10]`
Ngoài ra thì k cần phải space ra như này đâu